content/updates/2017-08-02-release-notes.md

```diff
@@ -0,0 +1,39 @@
++++
+date = "2017-08-02"
+title = "Platform Release Notes: August 2, 2017"
++++
+
+Curious what’s new that you might find helpful as a cloud.gov user? Here are highlights from our platform updates over the past two weeks.
+<!--more-->
+
+Curious what’s new that you might find helpful as a cloud.gov user? Here are highlights from our platform updates over the past two weeks.
+
+### Added
+You can [add people to spaces](https://cloud.gov/docs/apps/managing-teammates/#give-roles-to-a-teammate) from the [dashboard](https://dashboard.fr.cloud.gov/#/) if you’re an Org Manager. For example, this makes it easier for program managers to add developers to projects without using the command line tool.
+
+### Changed
+In the dashboard, Org Managers and Space Managers can [remove a person from a space](https://cloud.gov/docs/apps/managing-teammates/#space-users) with one click on "Remove All Space Roles".
+Org managers must remove a person from all spaces before removing them from the Org otherwise. If you try to remove them with the dashboard, you’ll see an error if the user still belongs to any space.
```

Comment:
trivial style thing: nice to stay consistent with capitalization for Org Managers